src/ripple/core/DatabaseCon.h

```diff
@@ -89,12 +89,17 @@ class DatabaseCon
         Config::StartUpType startUp = Config::NORMAL;
         bool standAlone = false;
         boost::filesystem::path dataDir;
+        static std::unique_ptr<std::vector<std::string> const> CommonPragma;
+        /// Shortcut used by the database connections that ignore the common
+        /// pragma strings
+        static const std::vector<std::string> NoCommonPragma;
```

Comment:
μNit: weird case for the name(s) here. Also confused as to why two separate things are needed.

Reply:
A lot of other static class variables have capitalized names, but I don't necessarily have to keep doing it.